package.json

```diff
@@ -1,20 +1,23 @@
 {
   "name": "node-vault",
   "version": "0.7.1",
-  "description": "Javascript client for HashiCorp's Vault",
-  "main": "./src/index.js",
+  "description": "NodeJS API client for HashiCorp's Vault",
+  "main": "./src/main.js",
   "scripts": {
-    "precoverage": "npm run lint",
-    "coverage": "istanbul cover --report text-summary node_modules/.bin/_mocha",
-    "lint": "eslint .",
+    "lint": "standard",
     "test": "mocha",
+    "coverage": "nyc --reporter text-summary mocha",
+    "nsp": "nsp check",
+    "complete-test": "yarn lint && yarn coverage",
+    "docker-test": "docker-compose build && docker-compose up --force-recreate test",
     "test-unit": "mocha test/unit.js",
     "test-watch": "mocha --watch",
     "predocs": "node scripts/write-features.js",
-    "docs": "docco --output docs src/*.js",
-    "nsp": "nsp check",
-    "prepublish": "npm run nsp",
-    "precommit": "node scripts/write-features.js && git add features.md"
+    "docs": "docco --output docs src/**/*.js",
+    "prepublishOnly": "yarn nsp",
+    "precommit": "node scripts/write-features.js && git add features.md",
+    "refresh-local-vault": "docker-compose down ; nohup docker-compose up vault &",
```

Comment:
Can you explain the purpose of this?

Reply:
I suppose you mean the `refresh-local-vault` task. It was a shortcut to refresh the vault setup fast, but I'm gonna remove it as it doesn't work very consistently and it is specific to the bash util `nohup` which might not be available in all environments. `nohup` is a bash utility that [lets you run a task in the background](https://unix.stackexchange.com/questions/137759/why-use-nohup-rather-than-exec).

Reply:
Fixed, I renamed it to `refresh-docker-vault` to make it more descriptive. Now it uses pure `docker-compose` commands: `docker-compose up --force-recreate vault`.

It is still possible to run it on the background by using `nohup yarn refresh-docker-vault &`.

Reply:
[The commit](https://github.com/DaniGuardiola/node-vault/commit/306751f24f54cb3471b94fbef6fef49b92f9f1a7#diff-b9cfc7f2cdf78a7f4b91a753d10865a2)